packages/network/src/logic/TrackerConnector.ts

```diff
@@ -62,4 +59,32 @@ export class TrackerConnector {
             this.maintenanceTimer = null
         }
     }
+
+    private connectTo({ id, ws }: TrackerInfo): void {
+        this.nodeToTracker.connectToTracker(ws, PeerInfo.newTracker(id))
+            .then(() => {
+                if (this.connectionStates.get(id) !== ConnectionState.SUCCESS) {
+                    logger.info('Connected to tracker %s', id)
+                    this.connectionStates.set(id, ConnectionState.SUCCESS)
+                }
+            })
+            .catch((err) => {
+                if (this.connectionStates.get(id) !== ConnectionState.ERROR) {
+                    // TODO we could also store the previous error and check that the current error is the same?
+                    // -> now it doesn't log anything if the connection error reason changes
+                    this.connectionStates.set(id, ConnectionState.ERROR)
+                    logger.warn('Could not connect to tracker %s, reason: %s', id, err.message)
+                }
+            })
+    }
+
+    private isActiveTracker(trackerId: TrackerId): boolean {
+        for (const streamKey of this.streamManager.getStreamKeys()) {
+            const { id: streamId, partition } = StreamIdAndPartition.fromKey(streamKey)
```

Comment:
`SPID`

Reply:
I'd rather we do the conversion from `StreamIdAndPartition` to `SPID` in a separate PR (to avoid having yet another standard -type of a situation). I made an issue on this https://linear.app/streamr/issue/NET-518/network-and-broker-replace-streamidandpartition-with-spid.